static/specification.html

```diff
@@ -301,24 +302,24 @@ <h2>
         </dt>
         <dd>
           The party receiving payments on behalf of the website, whose details
-          are provided by a [=payment end-point=].
+          are provided by a [=payment pointer=].
         </dd>
         <dt>
-          <dfn>Payment end-point</dfn>:
+          <dfn>Payment Pointer</dfn>:
```

Comment:
Given the explanation `A [=URL=] to an ...` and the use of `<link>` tag, could we use `Payment end-point` instead?

Also, for non-Interledger folks, `Payment Pointer` might be confusing and `Payment end-point` seems to be a better fit for a Web API

Reply:
What is a **payment end-point**?

A payment pointer is a well-defined concept and this definition aligns exactly with what it is.

Reply:
> A payment pointer is a well-defined concept

With a well defined syntax, so it's not exactly 'neat'